templates/resources/primary_connection.md.tmpl

```diff
@@ -19,7 +19,14 @@ description: |-
 ## Example Usage
 
 {{ tffile (printf "examples/resources/%s/resource.tf" .Name)}}
+
 -> **Note** Instead of using fully_qualified_name, you can reference objects managed outside Terraform by constructing a correct ID, consult [identifiers guide](https://registry.terraform.io/providers/Snowflake-Labs/snowflake/latest/docs/guides/identifiers#new-computed-fully-qualified-name-field-in-resources).
+
+-> **Note** To demote [`snowflake_primary_connection`](https://registry.terraform.io/providers/Snowflake-Labs/snowflake/latest/docs/resources/primary_connection) to [`snowflake_secondary_connection`](https://registry.terraform.io/providers/Snowflake-Labs/snowflake/latest/docs/resources/secondary_connection), resources need to be migrated manually. For guidance on removing and importing resources into the state check [resource migration](https://github.com/Snowflake-Labs/terraform-provider-snowflake/blob/main/docs/technical-documentation/resource_migration.md). Remove the resource from the state, then recreate it in manually using:
+    ```
+    CREATE CONNECTION <name> AS REPLICA OF <organization_name>.<account_name>.<connection_name>;
```

Comment:
Nit: in such blocks, MD formatting is a bit different, and this gets inlined. This is fine; I just write for future reference.

Reply:
I checked in https://registry.terraform.io/tools/doc-preview and this is how you can include more than one line in the note block

Reply:
Hmm, strange. I checked the docs/issues some time ago and it was not possible to have newlines in these notes/warnings.

Reply:
From: https://developer.hashicorp.com/terraform/registry/providers/docs#callouts, `You can't make multi-paragraph callouts.`

Reply:
https://registry.terraform.io/providers/Snowflake-Labs/snowflake/latest/docs/resources/primary_connection#example-usage This SQL got inlined.